src/types/index.ts

```diff
@@ -44,6 +44,8 @@ export type SearchResult = {
   displayTitle?: string | JSX.Element;
   tableOfContents?: string;
   wrapInCode?: boolean;
+
```

Comment:
Empty line?

Reply:
@ovflowd 

Just removed the empty line that was added by mistake. Thank you :D